mandatory/2-function-creation.js

```diff
@@ -16,7 +27,20 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (num > 100) { num = 100; }
+  if (Number.isInteger(num)) {
+    num = num;
+  }
+  else { num = num.toFixed(2); }
+      num = `${num}%`;
+  if (num.includes("0") && num.indexOf("0") === 4) {
+    num = num.slice(0, 4);
+    num = `${num}%`;
+
```

Comment:
This will only remove a single trailing zero. So it will change 18.10 to 18.1. But it won't change 18.00 to 18, for example. This is quite tricky, and there are a few different ways of doing it. Do some googling about removing trailing zeros in javascript, and decide which method is best.

Reply:
I've now changed it to parseFloat thanks, otherwise, I wouldn't have known about it. Thanks, much appreciated.
	num = parseFloat(num.toFixed(2));